core/cmd/local_client.go

```diff
@@ -339,13 +339,13 @@ func copyFile(src, dst string) error {
 	if err != nil {
 		return err
 	}
-	defer from.Close()
+	defer func() { _ = from.Close() }()
 
 	to, err := os.Create(dst)
 	if err != nil {
 		return err
 	}
-	defer to.Close()
+	defer func() { _ = to.Close() }()
```

Comment:
I dont really like this change. It doesn't buy as any additional safety and is much more difficult for the eye to parse.

Reply:
@samsondav what do you think of something like?
`defer logger.IfCallingError(to.Close)`